precompiles/erc20/integration_test.go

```diff
@@ -429,14 +437,15 @@ var _ = Describe("ERC20 Extension -", func() {
 				)
 
 				// Check that the allowance was removed since we authorized only the transferred amount
-				//
-				// TODO: adjust for original ERC20 calls
-				s.expectNoSendAuthz(spender.AccAddr, owner.AccAddr)
+				s.ExpectNoSendAuthzForContract(
+					callType, contractData,
+					spender.Addr, owner.Addr,
+				)
 			},
 				Entry(" - direct call", directCall),
 				// NOTE: we are not passing the contract call here because this test is for direct calls only
 
-				// FIXME: this fails currently because of the error setting up the allowance (see above)
+				// FIXME: other than the EVM extension, the ERC20 contract emits an additional Approval event (we only emit 1x Transfer)
 				Entry("- through erc20 contract", erc20Call),
```

Comment:
The ERC20 emits another event here when adjusting the allowance, we don't currently!

Reply:
Actually when comparing the newer OpenZeppelin v5 contract with our EVM extensions, the additional event is not emitted anymore. This seemed to have been the case only for the `ERC20MinterBurnerDecimals` contract we used.